src/hotspot/share/opto/phasetype.hpp

```diff
@@ -68,9 +68,9 @@
   flags(AFTER_RANGE_CHECK_ELIMINATION,  "After Range Check Elimination") \
   flags(BEFORE_PRE_MAIN_POST,           "Before Pre/Main/Post Loops") \
   flags(AFTER_PRE_MAIN_POST,            "After Pre/Main/Post Loops") \
-  flags(SUPERWORD1_BEFORE_SCHEDULE,     "Superword 1, Before Schedule") \
-  flags(SUPERWORD2_BEFORE_OUTPUT,       "Superword 2, Before Output") \
-  flags(SUPERWORD3_AFTER_OUTPUT,        "Superword 3, After Output") \
+  flags(AUTO_VECTORIZE1_BEFORE_APPLY,   "AutoVectorization 1, Before Apply") \
+  flags(AUTO_VECTORIZE2_AFTER_REORDER,  "AutoVectorization 2, Before Apply Memop Reordering") \
+  flags(AUTO_VECTORIZE3_AFTER_APPLY,    "AutoVectorization 3, After Apply") \
```

Comment:
Note: renamed them since I had to move them anyway from `SuperWord` -> `VTransform`.